src/app/review-rating/review-rating.component.html

```diff
@@ -10,7 +10,7 @@ <h1 class="title">Rating</h1>
   <h2 class="title">How helpful was the feedback?</h2>
 
   <div>
-    <ion-range min="0" max="1" step="any" [(ngModel)]="ratingData.rating"></ion-range>
+    <ion-range mode="ios" min="0" max="1" step="0.1" snaps="true" [(ngModel)]="ratingData.rating" (ngModelChange)="roundUpRating()"></ion-range>
```

Comment:
we can just round up the rating when we post the data, don't need to do the round up every time when the value changes

Reply:
yes. you are correct. and we already have that. what i add is a extra thing. it can removed from the code. I will update the code.